Dragging handles of InteractiveGaussian figure

grtlr opened the pull request:
Fixes the z-order so that dragging the handles also works if the handle was clicked directly.

Related issues are #29 and #43.

@ludwigschubert: This PR should provide a quick fix to the problem. In this state, however, it might not handle hovering correctly. Unfortunately, I can't run the article locally anymore (using `npm run serve`). The figures don't load anymore. Could you please try to reproduce this problem, if you find time? Thank you!

ludwigschubert commented:
OK! I found a couple more places where the handles needed to be moved to the top. I also fixed the loading issue (we add polyfills during deployment to Distill.pub), stopped the header from reflowing when loading, and slimmed the JS bundle a bunch to make loading the article faster.

grtlr commented:
Great—thank you very much! Initially I added the source maps for the review phase and then forgot to take the out for the actual release, now that all the source code is openly available.

> On 9. Apr 2019, at 00:09, Ludwig Schubert <notifications@github.com> wrote:
> 
> OK! I found a couple more places where the handles needed to be moved to the top. I also fixed the loading issue (we add polyfills during deployment to Distill.pub), stopped the header from reflowing when loading, and slimmed the JS bundle a bunch to make loading the article faster.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
